.github/workflows/bindings.yml

```diff
@@ -0,0 +1,71 @@
+name: FFI-CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+env:
+  RUST_BACKTRACE: 1
+  RUSTFLAGS: -Dwarnings
+  RUSTDOCFLAGS: -Dwarnings
+  MSRV: "1.66"
+
+jobs:
+  build_and_test_ffi_bindings:
+    name: Build and test ffi bindings
+    runs-on: ${{ matrix.name }}
+    strategy:
+      fail-fast: false
+      matrix:
+        name: [macOS-latest]
+        rust: [nightly, stable]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+        with:
+          submodules: recursive
+
+      - name: Install ${{ matrix.rust }}
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ matrix.rust }}
+
+      - name: Create a copy of iroh.h
+        run: |
+          cd iroh-ffi
+          cp iroh.h iroh.compare.h
```

Comment:
alternatively you can use git diff

Reply:
Eh, since `iroh.h` is in `.gitignore` it doesn't seem to diff it. Also think cmp is slightly faster.